File: internal/collector/treesitter/testdata/test_kind/javascript/js-jest/test/login.test.js
// SPDX-License-Identifier: Apache-2.0

describe("login", () => {
  it("succeeds", () => {
    expect(1 + 1).toBe(2);
  });

  test.each([
    [1, 1, 2],
    [2, 3, 5],
  ])("adds %i + %i = %i", (a, b, want) => {
    expect(a + b).toBe(want);
  });
});

File: internal/collector/treesitter/testdata/test_kind/ocaml/ocaml-alcotest/test/login_test.ml
(* SPDX-License-Identifier: Apache-2.0 *)

let test_login () =
  Alcotest.(check int) "adds" 2 (1 + 1)

let () =
  Alcotest.run "login" [
    "basic", [ Alcotest.test_case "login" `Quick test_login ];
  ]
